samples/grids/list/styling/App.razor

```diff
@@ -2,27 +2,46 @@
 @inject IIgniteUIBlazor IgniteUIBlazor
 
 <div class="container vertical">
+    <style>
+        igc-list-header {
+            font-size: 20px;
```

Comment:
please always use 'rem' units by dividing all 'px' values by 16px which is 1rem, e.g.
20px / 16px = 1.25rem

```font-size: 1.25rem;```